test/fixtures/simple_example/outputs.tf

```
@@ -14,12 +14,32 @@
  * limitations under the License.
  */
 
-output "bucket_name" {
-  description = "The name of the bucket."
-  value       = "${module.example.bucket_name}"
+output "ids" {
```

The name should indicate that this is a map rather than a list.

```suggestion
output "names_and_ids" {
```